homeassistant/components/yamaha/media_player.py

```diff
@@ -205,6 +205,8 @@ def __init__(self, name, receiver, source_ignore, source_names, zone_names):
         self._play_status = None
         self._name = name
         self._zone = receiver.zone
+        if self.receiver.serial_number is not None:
```

Comment:
```suggestion
        if self.receiver.serial_number is not None:
            # Since not all receivers will have a serial number and set a unique id
            # the default name of the integration may not be changed
            # to avoid a breaking change.
```